include/statespace/SE2.hpp

```diff
@@ -146,6 +146,17 @@ class SE2 : public virtual StateSpace {
     /// Documentation inherited
     double get_resolution() const override;
 
+    /// Adds two dynamic Eigen vectors together
+    ///
+    /// \param _vector_1 The first vector to add
+    /// \param _vector_2 The second vector to add
+    /// \param[out] _vector_out Output vector
+    /// \return True if calculation successful, false otherwise
```

Comment:
True if calculation successful, false otherwise (i.e. input vectors must be the same size)